vue-components/src/components/DateInput.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+	<div
+		:class="[ 'wikit', 'wikit-DateInput' ]"
+	>
+		<InputWithExtender
+			:label="label"
+			:value="value"
+			:error="error"
+			:placeholder="placeholder"
+			:disabled="disabled"
+			@input="$emit( 'input', $event )"
+		>
+			<template v-slot:default>
+				<p class="wikit-DateInput__IntroText">
+					{{ resultsIntroText }}
+				</p>
+				<p class="wikit-DateInput__ParsedValue" v-if="parsedValue">
+					<b>{{ parsedValue }}</b>
+				</p>
+				<slot v-if="!value" name="prompt" />
+				<BouncingDots v-if="!parsedValue && value" type="small" />
+				<div class="wikit-DateInput__CalendarNotice" v-if="calendarNotice && parsedValue">
+					<Icon type="info" size="small" />
```

Comment:
can we apply `color="subtle"` to the icon or make it inherit the color of the text it accompanies? Any of the options are fine

Reply:
Done

Reply:
We don't have "subtle" color type. The ones are `'base', 'warning', 'error', 'notice', 'success', 'inherit'`. I change it to inherit.

Reply:
thanks! inherit makes sense. I'm opening a small PR to add those missing colors (I was convinced they were there, but maybe mixed up colors and sizes), but no further change is needed here.